docs/browser.md

```diff
@@ -57,3 +57,23 @@ To get to the AWS Toolkit logs:
 
 1. Open Command Palette: `cmd/ctrl` + `shift` + `p`
 2. Type: `AWS: View Toolkit Logs`
+
+## Finding incompatible transitive dependencies
+
+For example, if I have a Typescript module, `myFile.ts`, that imports a module, which imports another module (transitive dependency) such as `fs-extra`,
+when I execute `myFile.ts` in the browser it will break due to `fs-extra` not being browser compatible.
+
+It may be difficult to determine which module imported `fs-extra` due to a nested chain of transitive dependencies.
+
+As a solution, we can use [`dependency-cruiser`](https://www.npmjs.com/package/dependency-cruiser) to generate a dependency diagram
+to help us visualize the imports and determine which module is importing a certain module.
+
+### How to use
+
+0. Ensure you have run `npm install --save-dev`
```

Comment:
why is `--save-dev` needed? `npm install` should install dev deps by default

Reply:
Since none of our modules are importing `dependency-cruiser`, can we change this instruction to:

    npm install dependency-cruiser

Then our `package-lock.json` won't have the extra deps, and we won't need to maintain that dep.